src/main/java/seedu/address/logic/commands/EditEntryCommand.java

```diff
@@ -0,0 +1,217 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_ENTRY_DOES_NOT_EXIST;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_END_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_START_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_ENTRIES;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.entry.Entry;
+import seedu.address.model.entry.EntryDate;
+import seedu.address.model.entry.EntryName;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Edits the details of an existing entry in the address book.
+ */
+public class EditEntryCommand extends Command {
+
+    public static final String COMMAND_WORD = "eedit";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the entry identified "
+            + "by the entry name used in the displayed entries list. "
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: ENTRY_NAME "
+            + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_START_DATE + "START_DATE] "
+            + "[" + PREFIX_END_DATE + "END_DATE] "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " Meeting "
+            + PREFIX_NAME + "Meeting with group ";
+
+    public static final String MESSAGE_EDIT_ENTRY_SUCCESS = "Edited entry: %1$s";
```

Comment:
i was thinking of moving constants to commons.core.Messages, going to do the same for #94

Reply:
Sure sounds good